.github/workflows/check_pr_target.yml

```diff
@@ -0,0 +1,25 @@
+name: Check PR target branch
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened, ready_for_review]
+
+permissions:
+  contents: read
+  pull-requests: read
+
+jobs:
+  check-pr:
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+    - name: Fail if not targeting nightly branch
+      if: ${{ github.base_ref != 'nightly' }}
+      run: |
+        echo "PRs must be targeted to merge to the nightly branch!"
+        echo "PR is currently targeting: \"${{ github.base_ref }}\""
```

Comment:
Commenting on PR might reduce turnaround time 
https://github.com/orgs/community/discussions/69944#discussioncomment-7269357

`- name: Create Comment
      if: steps.pr-changes.outputs.changes != '[]'
      run: |
        # Use GitHub API to create a comment on the PR
        PR_NUMBER=${{ github.event.pull_request.number }}
        COMMENT="Your checklist comment here"
        GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
        COMMENT_URL="https://api.github.com/repos/${{ github.repository }}/issues/${PR_NUMBER}/comments"

        curl -s -H "Authorization: token ${GITHUB_TOKEN}" -X POST $COMMENT_URL -d "{\"body\":\"$COMMENT\"}"`

Reply:
Hi MertCelik, thanks for the suggestion!

The `pr-lint-action` we depend on actually has a setting to configure this behavior, which I've now enabled.